carbon/assetlist.json

```diff
@@ -61,67 +52,33 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.svg"
       },
-      "images": [
-        {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.png",
-          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.svg",
-          "theme": {
-            "primary_color_hex": "#053e4f"
-          }
-        }
-      ]
+      "coingecko_id": "usd-coin"
     },
     {
-      "description": "BNB token on Carbon",
+      "description": "Grouped USD on Carbon",
       "denom_units": [
         {
-          "denom": "bnb.1.6.773edb",
+          "denom": "cgt/1",
           "exponent": 0
         },
         {
-          "denom": "bnb",
+          "denom": "usd",
           "exponent": 18,
           "aliases": [
-            "BNB"
+            "USD"
           ]
         }
       ],
-      "base": "bnb.1.6.773edb",
-      "name": "Binance Coin",
-      "display": "bnb",
-      "symbol": "BNB",
-      "traces": [
-        {
-          "type": "bridge",
-          "counterparty": {
-            "chain_name": "binancesmartchain",
-            "base_denom": "wei",
-            "contract": "0xb5D4f343412dC8efb6ff599d790074D0f1e8D430"
-          },
-          "chain": {
-            "contract": "0xb5D4f343412dC8efb6ff599d790074D0f1e8D430"
-          },
-          "provider": "PolyNetwork"
-        }
-      ],
+      "type_asset": "sdk.coin",
+      "base": "cgt/1",
+      "name": "Demex Grouped USD",
+      "display": "usd",
+      "symbol": "USD",
       "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/binancesmartchain/images/bnb.png",
-        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/binancesmartchain/images/bnb.svg"
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/carbon-grouped-usd.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/carbon-grouped-usd.svg"
       },
-      "images": [
-        {
-          "image_sync": {
-            "chain_name": "binancesmartchain",
-            "base_denom": "wei"
-          },
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/binancesmartchain/images/bnb.png",
-          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/binancesmartchain/images/bnb.svg",
-          "theme": {
-            "primary_color_hex": "#f3bb0c"
-          }
-        }
-      ],
-      "type_asset": "sdk.coin"
+      "coingecko_id": "usd-coin"
```

Comment:
This coingecko ID doesn't belong with Carbon Grouped USD. There would need to be a qualifying path tracing to the original USDC, which I don't think this would be closely related enough